### DISCUSSION TITLE
Welcome to Zillacode Discussions!

### DISCUSSION THREAD

**Discussion by davidzajac1** [davidzajac1/zillacode#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shankar1998** [comment]
can i get the location of pyspark ques and ans location, so that i can practice in databricks

**Reply by davidzajac1** [reply]
@shankar1998 they are stored here in this Python file, but they aren't stored in a manner that's easy to use, there's a lot of other functions that parse the strings, etc. to turn them into what you see in the UI: https://github.com/davidzajac1/zillacode/blob/master/backend/constants.py#L33

**Comment by Vijayreddy81** [comment]
got below error. Please help.

$ docker compose up
unable to get image 'davidzajac1/zillacode-spark:latest': unexpected end of JSON input

**Reply by davidzajac1** [reply]
@Vijayreddy81 can you post a screenshot of this here please?
